docs/guide/exe_verilator.rst

```diff
@@ -1193,13 +1193,13 @@ Summary:
    by size (plain :vlopt:`--stats` just gives a count).  See
    :vlopt:`--stats`, which is implied by this.
 
-.. option:: --structs-packed
+.. option:: --structs-unpacked
```

Comment:
```suggestion
.. option:: --structs-packed
```

Reply:
I modified this line to mention `--structs-unpacked` because I think the documentation is a bit misleading here.
Comments and docs do mention the `--structs-packed` flag, but it was never parsed by Verilator, there's only `--structs-unpacked`.

Now I'm a little confused about this: do we want to decide on one of these flags or have both?
I think keeping both is error prone.

Reply:
Oh, I missed that, you are right the old code had --structs-unpacked, though the variable was incorrectly called structsPacked, you are right that is very confusing.  Let's go with your original thought, rename the flag to --structs-packed, make the variable m_structsPacked now defaulting to false, and update docs appropriately.

BTW for other items that are done please hit "resolve conversation" thanks.